docs/useCreateController.md

```diff
@@ -43,24 +43,24 @@ export const BookCreate = () => {
 
 **Tip**: If you just use the return value of `useCreateController` to put it in an `CreateContext`, use [the `<CreateBase>` component](./CreateBase.md) instead for simpler markup.
 
-`useCreateController` accepts an options argument, with the following fields, all optional:
+## Arguments
```

Comment:
We use Parameters in most hooks. Why is Arguments better? And if you change it here, you must change it everywhere.

Reply:
Wasn't sure what's the preference, both work for me. Will change it back soon!